app/interactors/create_temporary_stripe_credit_card.rb

```diff
@@ -63,7 +61,8 @@ def create_stripe_card_bank_account(org, stripe_tok, card_params)
         stripe_customer_id: org.stripe_customer_id,
         stripe_tok: stripe_tok
       )
-      bank_account = org.bank_accounts.create(card_params.merge(stripe_id: card.id))
+      bank_account = org.bank_accounts.create!(card_params.merge(stripe_id: card.id))
```

Comment:
@thermistor note the bang. Seemed like a good idea.

Reply:
I like bangs generally, you want to know if things fail unexpectedly.